src/app/public/modules/date-range-picker/date-range-picker.component.scss

```diff
@@ -8,9 +8,9 @@
 }
 
 .sky-date-range-picker-form-group {
-  flex: 1 1 auto;
   padding-left: $sky-padding-half;
   padding-right: $sky-padding-half;
+  flex-basis: calc(100% / 3);
```

Comment:
Should we have a mobile version of this style in the `@media` block below?
```
.sky-date-range-picker-form-group {
  flex-basis: 100%;
  ...
```

Reply:
`flex-basis` *should* change the vertical height when you have `flex-direction` set to `column`. Strange it's not having that effect though... Anyways, it does work and doesn't need to hold this up. I'm ok with what you've got.

Reply:
if it's safer to add that we can, doing so doesn't seem to break anything

Reply:
Yeah, lets do it. I think it's better to be more deliberate in this case.